src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -1046,6 +1046,13 @@ Not all possible configuration options may be set via environment variables. The
           <td></td>
           <td></td>
         </tr>
+        
+          <tr>
+          <td>`Service.version`</td>
+          <td></td>
+          <td>`NEW_RELIC_METADATA_RELEASE_TAG`</td>
```

Comment:
This doesn't look right since the tags.service.version attribute is set by `NEW_RELIC_METADATA_SERVICE_VERSION`
The collapser is just here to map the environment variable with the config option described elsewhere in this doc.  Since our environment variable is not described elsewhere in the page I think we should add a section for version tracking